app/Base/index.tsx

```diff
@@ -32,9 +32,11 @@ import {
 } from '#base/utils/restRequest';
 import 'mapbox-gl/dist/mapbox-gl.css';
 
+import { mapboxToken } from '#base/configs/env';
+
 import styles from './styles.css';
 
-setMapboxToken(process.env.REACT_APP_MAPBOX_ACCESS_TOKEN as string);
+setMapboxToken(mapboxToken as string);
```

Comment:
We should not cast mapbox token.
Let's set mapbox token if mapboxToken is defined